test/functionalTest/cases/1715_batch_update/batch_update_errors.test

```diff
@@ -0,0 +1,273 @@
+payload='{
+  "actionType": "UPDATE",
+  "entities": [
+    {
+      "id": "E",
+      "type": "T",
+      "foo": 9
+    }
+  ]
+}'
+orionCurl --url /v2/op/update --payload "$payload" --json
+echo
+echo
+
+
+--REGEXPECT--
+01. Batch Update without payload
+================================
+HTTP/1.1 411 Length Required
+Content-Length: 91
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Zero/No Content-Length in PUT/POST/PATCH request",
+    "error": "LengthRequired"
+}
+
+
+02. Batch Update with empty payload
+===================================
+HTTP/1.1 400 Bad Request
+Content-Length: 52
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "empty payload",
+    "error": "BadRequest"
+}
+
+
+03. Batch Update with invalid JSON payload
+==========================================
+HTTP/1.1 400 Bad Request
+Content-Length: 75
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Errors found in incoming JSON buffer",
+    "error": "ParseError"
+}
+
+
+04. Batch Update without 'actionType' (which is mandatory)
+==========================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 99
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Invalid JSON payload, mandatory field /actionType/ not found",
+    "error": "BadRequest"
+}
+
+
+05. Batch Update without 'entities' (which is mandatory)
+========================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 97
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Invalid JSON payload, mandatory field /entities/ not found",
+    "error": "BadRequest"
+}
+
+
+06. Batch Update with unrecognized item 'foo'
+=============================================
+HTTP/1.1 400 Bad Request
+Content-Length: 80
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Unrecognized field in JSON payload: /foo/",
+    "error": "BadRequest"
+}
+
+
+07. Batch Update with empty entity vector
+=========================================
+HTTP/1.1 400 Bad Request
+Content-Length: 75
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Errors found in incoming JSON buffer",
+    "error": "ParseError"
+}
+
+
+08. Batch Update with entity without 'id'
+=========================================
+HTTP/1.1 400 Bad Request
+Content-Length: 71
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "nor /id/ nor /idPattern/ present",
+    "error": "BadRequest"
+}
+
+
+09. Batch Update with entity with unknown field (and no options=keyValues)
+==========================================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 70
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "attribute must be a JSON object",
+    "error": "BadRequest"
+}
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB
```

Comment:
As far as I checked (maybe I have missed something) all the /op/update cases use only one entity in the payload. It would be interesesting to add some test using more than one (take into account that the "value" of update batch is to modify/add/delete several entities in a single operation, for only-one-entity operation, the usual REST operations could be used).